mirrormanager2/default_config.py

```diff
@@ -78,6 +78,13 @@
 # Default: ``.MirrorManager``.
 MM_COOKIE_NAME = 'MirrorManager'
 
+# If this variable is set (and the directory exists) the crawler
+# will create per host log files in MM_LOG_DIR/crawler/<hostid>.log
+# which can the be used in the web interface by the mirror admins.
```

Comment:
No, I had to change the SELinux context of that directory on the staging crawler.